versioned_docs/version-v3.2/documentation-generation/docs-readme.md

```diff
@@ -28,16 +33,28 @@ stencil build --docs
 ```
 
 This will cause the Stencil compiler to perform a one-time generation of README
-files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output target.
+files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output
```

Comment:
Doesn't this just link to itself?

Reply:
lol it definitely does! I'll get rid of that while we're in here.